packages/yew-macro/tests/hook_macro/use_prepared_state-fail.stderr

```diff
@@ -26,17 +28,19 @@ error: this hook takes 2 arguments but 1 argument was supplied
    |
    = note: this error originates in the macro `use_prepared_state_with_closure` (in Nightly builds, run with -Z macro-backtrace for more info)
 
-error: expected closure
-  --> tests/hook_macro/use_prepared_state-fail.rs:19:41
+error: this hook takes 2 arguments but 1 argument was supplied
+  --> tests/hook_macro/use_prepared_state-fail.rs:19:5
    |
 19 |     use_prepared_state_without_closure!(123)?;
-   |                                         ^^^
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: this error originates in the macro `use_prepared_state_without_closure` (in Nightly builds, run with -Z macro-backtrace for more info)
 
-error: You must specify a return type for this closure. This is used when the closure is omitted from the client side rendering bundle.
-  --> tests/hook_macro/use_prepared_state-fail.rs:21:41
+error: expected closure
+  --> tests/hook_macro/use_prepared_state-fail.rs:21:58
    |
 21 |     use_prepared_state_without_closure!(|_| { todo!() }, 123)?;
-   |                                         ^^^^^^^^^^^^^^^
+   |                                                          ^^^
```

Comment:
Similar to above, this testing is asserting that the closure has specified a return type.